src/rapids_dependency_file_generator/schema.json

```diff
@@ -0,0 +1,160 @@
+ {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "https://raw.githubusercontent.com/rapidsai/dependency-file-generator/v1.0.0/src/rapids_dependency_file_generator/schema.json",
+    "type": "object",
+    "title": "RAPIDS Package Dependency Specification Format",
+    "description": "Consolidated specification of RAPIDS project dependencies.",
```

Comment:
```suggestion
    "description": "Consolidated specification of RAPIDS project dependencies",
```

Reply:
It's not a sentence, is it? 😕

Reply:
Correct, it is only a phrase.

Reply:
Classic _confirmation bias_. I am so used to arguing against the dot after non-sentences/phrases that I misread the diff. 🤦🏻

Reply:
Done.